birdhouse/env.local.example

```diff
@@ -52,9 +52,19 @@ export POSTGRES_MAGPIE_PASSWORD=postgres-qwerty
 # * Assemble different combinations of components/functionalities by including
 #   only the config/docker-compose fragment necessary.
 #
-# Last dir/docker-compose fragment in the list have highest override precedence.
-# Ex: last docker-compose volume mount to same destination win over the
-# previous docker-compose volume mount.
+# Last dir/component in the EXTRA_CONF_DIRS list have highest override
+# precedence, example:
+#
+#   * Last docker-compose volume mount to same destination win over the
+#     previous docker-compose volume mount.
+#
+#   * Last default.env can change the values of all previous default.env.
+#
+#   * Last pre/post docker-compose-up script can potentially undo actions
+#     from previous scripts.
+#
+# Suggested to keep the private-config-repo last in the list to be able to
+# override anything.
```

Comment:
Thanks for specifying. 
It is worth mentioning since with the pre/post hooks and custom configs, precedence can become tricky at a certain point.

Reply:
Agreed.  The variable names in each component `default.env` file should not collide else one will override the other one.  Probably not an issue for components in this repo but if we pull in components from other repos, this might become an issue.